lib/gis_robot_suite.rb

```diff
@@ -143,6 +143,7 @@ def self.locate_data_files(dir)
     spatial_index_x_shapefile = Dir.glob("#{dir}/*.sbx").first
     projection_file = Dir.glob("#{dir}/*.prj").first
     coding_file = Dir.glob("#{dir}/*.cpg").first
+    json_file = Dir.glob("#{dir}/*.json").first
```

Comment:
Updated to pick up `.json` files, including `index_map.json`, so they can be renamed as `.geojson`. The only other place this method is currently used is the `copy-data` step, which copies from `temp` to `content` and will eventually be removed.